cli/cli.go

```diff
@@ -65,17 +65,9 @@ func New(version string) *CLI {
 // IsUpToDate compares the current version to that of the latest release.
 func (c *CLI) IsUpToDate() (bool, error) {
 	if c.LatestRelease == nil {
-		resp, err := HTTPClient.Get(LatestReleaseURL)
-		if err != nil {
-			return false, err
-		}
-		defer resp.Body.Close()
-
-		var rel Release
-		if err := json.NewDecoder(resp.Body).Decode(&rel); err != nil {
+		if err := c.fetchLatestRelease(); err != nil {
```

Comment:
I ended up moving this into a separate method, but I can't remember if I had a good reason or not.